cognite/neat/workflows/steps/lib/graph_loader.py

```diff
@@ -321,10 +321,10 @@ class GenerateCDFRelationshipsFromGraph(Step[CategorizedRelationships]):
 
     def run(  # type: ignore[override]
         self, rules: RulesData, cdf_client: CogniteClient, solution_graph: SolutionGraph
-    ) -> (FlowMessage, CategorizedRelationships):
+    ) -> tuple[FlowMessage, CategorizedRelationships]:
```

Comment:
Did we agree that this should be? I think Aleks used that in the frontend. 
```suggestion
    ) -> (FlowMessage, CategorizedRelationships):
```

Reply:
we did, probably I made mistake when merging main changes